toqito/matrix_props/mutual_coherence.py

```diff
@@ -0,0 +1,61 @@
+"""Computes the mutual coherence of the columns of a matrix or a list of 1D numpy arrays."""
```

Comment:
The description here says that it operates on a list of 1D arrays, but the input variable name is called "matrix" which I think is incorrect/misleading.